errors.go

```diff
@@ -279,8 +279,10 @@ func (w *withMessage) Format(s fmt.State, verb rune) {
 			return
 		}
 		fallthrough
-	case 's', 'q':
+	case 's':
 		io.WriteString(s, w.Error())
+	case 'q':
+		fmt.Fprintf(s, "%q", w.Error())
```

Comment:
It would be more efficient to directly call [`strconv.AppendQuote`](https://pkg.go.dev/strconv#AppendQuote):

```go
s.Write(strconv.AppendQuote(nil, w.Error()))
``` 
```